content/blog/five-git-tricks.md

```diff
@@ -0,0 +1,199 @@
++++
+title = "Five Git Tricks To Make Crafting Software Easier"
+slug = "five-git-tricks"
+date = 2022-06-14T00:10:46+05:30
+image = "/images/2022/06/octocat.jpg"
+draft = true
+author = "Abhishek Keshri"
+description = ""
+tags = ["Git", "Software Craftsmanship"]
+categories = ["Git", "Software Craftsmanship"]
+type = ""
++++
+
+
+Git is the most widely used version control system in the world and it is something that every Software Craftsperson should master.
+
+In this article, we'll be going through five git tricks that will make your software crafting easier.
+
```

Comment:
Would be nice to add a list of the 5 tricks upfront and then dive deeper into each